website/src/pages/tr/index.json

```
@@ -1,99 +1,99 @@
 {
-  "title": "Ana sayfa",
-  "hero": {
-    "title": "The Graph Docs",
-    "description": "Kick-start your web3 project with the tools to extract, transform, and load blockchain data.",
-    "cta1": "How The Graph works",
-    "cta2": "Build your first subgraph"
-  },
-  "products": {
-    "title": "The Graph’s Products",
-    "description": "Choose a solution that fits your needs—interact with blockchain data your way.",
-    "subgraphs": {
-      "title": "Subgraph'ler",
-      "description": "Extract, process, and query blockchain data with open APIs.",
-      "cta": "Develop a subgraph"
+    "title": "Ana Sayfa",
+    "hero": {
+        "title": "The Graph Belgeleri",
+        "description": "Web3 projenizi başlatmak için blok zinciri verilerini çıkarıp, dönüştürmek ve yüklemek için araçlar.",
+        "cta1": "The Graph nasıl çalışır",
+        "cta2": "İlk alt grafiğinizi oluşturun"
     },
-    "substreams": {
-      "title": "Substream'ler",
-      "description": "Fetch and consume blockchain data with parallel execution.",
-      "cta": "Develop with Substreams"
+    "products": {
+        "title": "The Graph Ürünleri",
+        "description": "İhtiyaçlarınıza uygun bir çözüm seçin — blok zinciri verileriyle etkileşime girin.",
+        "subgraphs": {
+            "title": "Alt Grafikler",
+            "description": "Açık API’lerle blok zinciri verilerini çıkarın, işleyin ve sorgulayın.",
+            "cta": "Bir alt grafiği geliştirin"
+        },
+        "substreams": {
+            "title": "Alt Akımlar",
+            "description": "Paralel yürütmeyle blok zinciri verilerini alın ve tüketin.",
+            "cta": "Alt Akımlarla Geliştirin"
+        },
+        "sps": {
+            "title": "Alt Akımlar Güçlü Alt Grafikler",
+            "description": "Alt Akımları kullanarak alt grafiğinizin verimliliğini ve ölçeklenebilirliğini artırın.",
+            "cta": "Alt Akımlar güçlü bir alt grafiği kurun"
+        },
+        "graphNode": {
+            "title": "Grafik Düğümü",
```

```suggestion
            "title": "Graph Node",
```

Graph Node is a product name